lib/encoder/encoder.go

```diff
@@ -190,7 +190,7 @@ func (mask *MultiEncoder) Set(in string) error {
 		if bits, ok := nameToEncoding[part]; ok {
 			out |= bits
 		} else {
-			i, err := strconv.ParseInt(part, 0, 64)
+			i, err := strconv.ParseUint(part, 0, 32)
```

Comment:
This is another 32 vs 64 bit problem with uint

Not sure what the correct solution here is as there is no `strconf.Atou`

Rlone will go wrong on 32 bit platforms if there are more than 32 options though so we could make MultiEncoder a `uint32` or `uint64` though I expect that might be quite a distruptive change!

Reply:
`strconv.ParseUint(part, 0, 0)` does the same.